recipes/r-immucellaimouse/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set version = "1.0" %}
+
+package:
+  name: r-immucellaimouse
+  version: {{ version }}
+
+source:
+  url:
+    https://github.com/lydiaMyr/ImmuCellAI-mouse/archive/refs/heads/main.zip
+  sha256: f83fb931ca2dc401d8d569f96620c709613934e4817ef12193b13f5cdf60d154
+
+build:
+  run_exports:
+    - {{ pin_subpackage('r-immucellaimouse', max_pin="2") }}
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  host:
+    - r-base
+    - r-roxygen2 >=6.0.1
+    - r-rcpp >=0.12.14
+    - bioconductor-gsva
+    - r-dplyr
+  run:
+    - r-base
+    - r-rcpp >=0.12.14
+    - bioconductor-gsva
+    - r-dplyr
+    - r-mass
+
+test:
+  commands:
+    - $R -e "library(ImmucellAImouse)"
```

```suggestion
    - $R -e "library(ImmuCellAImouse)"
```

I think this is case sensitive and should be spelled this way?